hw10/style.css

```diff
@@ -0,0 +1,130 @@
+@font-face {
+    font-family: Raleway;
+    src: url(Raleway/Raleway-VariableFont_wght.ttf);
+}
+body{
+    margin: 0;
+    height: 100vh;
+    background: linear-gradient(rgb(134 64 207 / 80%), #b3858d61);
+}
+.header{
+    color: white;
+    font-family: Raleway;
+    font-size: 60px;
+    text-align: center;
+}
+.text__basic{
+    font-family: Raleway;
+    color: rgba(255, 255, 255, 0.685);
+    text-align: center;
+    font-size: 12px;
+    word-break: break-all;
+}
+.gallery{
+    display: flex;
+    justify-content: center;
+}
+.gallery-center{
+    display: flex;
+    margin-left: 20px;
+    margin-right: 20px;
+    box-shadow: 0px 4px 4px 0px rgb(93 20 169 / 47%);
+}
+.gallery-right{
+    display: flex;
+    box-shadow: 0px 4px 4px 0px rgb(93 20 169 / 47%);
+}
+.gallery-left{
+    display: flex;
+    box-shadow: 0px 4px 4px 0px rgb(93 20 169 / 47%);
+}
+.gallery-left:hover{
+    transform: scale(1.05);
+}
+.gallery_img:hover{
+    filter: blur(2px);
+}
+.gallery-center:hover{
+    transform: scale(1.05);
+}
+.gallery-right:hover{
+    transform: scale(1.05);
+}
+.gallery__headline{
+    position: absolute;
+    color: white;
+    font-family: Raleway;
+    width: 197px;
+    margin-top: 145px;
+    margin-left: 15px;

```

Comment:
If you are using position: absolute, lets use "top:" and "left:" instead of "margin-top:" and "margin-left:"